scripts/run.js

```diff
@@ -6,22 +6,30 @@ const { exec, execSync } = require("child_process");
 const PORT = 5050;
 const systemType = os.platform();
 
-if (systemType !== "darwin") {
-  console.log("This command currently only supports unix at the moment.");
-  return;
-}
-
 function newTerminal(command) {
-  const PWD = path.join(__dirname, "../");
-  execSync(`
-  osascript -e 'tell application "Terminal" to activate' \
-  -e 'tell application "System Events" to keystroke "t" using {command down}' \
-  -e 'tell application "Terminal" to do script "cd ${PWD} && ${command}" in front window'
-  `);
+  const CWD = path.join(__dirname, "../");
+
+  // empty double quotes is intentional because of how the "start" Windows
+  // command parameters work
+  if (systemType === "win32") {
+    setTimeout(() => {
+      exec(`start "" /d ${CWD} ${command}`);
+    }, 1000);
```

Comment:
Why is `setTimeout` needed here?

Reply:
I was running into a problem while testing where having just the `exec()` statements would open terminals too fast and skip the commands in a sense. The `ngrok HTTP PORT` command would not be executed while the new terminal windows for `yarn app:start` and `yarn server:start` opened fine. Adding a one second delay with `setTimeout()` fixed that problem for me.